### DISCUSSION TITLE
Cannot change dependency version from default of lowest.

### DISCUSSION THREAD

**Discussion by FaustoNascimento** [NuGet/Home#11882] [General]
Using Visual Studio 2022 17.2.3, trying to install `protobuf-net.Grpc`, which lists the following dependencies (among others):

 `Grpc.Core.Api >= 2.36.1`
`protobuf-net >= 2.4.6`

`Grpc.Core.Api`'s current version is 2.46.3 and the current version for `protobuf-net` is 3.1.4.

I would like the latest dependencies to be installed, not the default of lowest version.

Using `Install-Package protobuf-net.Grpc -DependencyVersion Highest` doesn't do anything at all. I've made sure that my `Default package management format` is set to `Packages.config` in Visual Studio 2022 and I've also tried modifying the `%appdata%\NuGet\NuGet.Config` file, currently it reads as follows:

```
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
  </packageSources>
  <packageRestore>
    <add key="enabled" value="True" />
    <add key="automatic" value="True" />
  </packageRestore>
  <bindingRedirects>
    <add key="skip" value="False" />
  </bindingRedirects>
  <packageManagement>
    <add key="format" value="0" />
    <add key="disabled" value="False" />
  </packageManagement>
  <config>
    <add key="dependencyversion" value="Highest" />
  </config>
</configuration>
```

Yet no matter what I try NuGet always downloads the lowest version dependencies, how do I change this behaviour?

**Comment by nkolev92** [comment]
In PackageReference, only 1 algorithm is available: https://github.com/NuGet/Home/issues/5553 covers the `highest` algorithm ask.
